lib/bottom_navy_bar.dart

```diff
@@ -47,17 +67,24 @@ class BottomNavyBar extends StatelessWidget {
         color: bgColor,
         boxShadow: [
           if (showElevation)
-            const BoxShadow(
-              color: Colors.black12,
-              blurRadius: 2,
+            BoxShadow(
+              color: Colors.black,
```

Comment:
It's better to have `BoxShadow.color` or` ‍‍‍‍‍BoxShadow` as constructor option.

Reply:
Right, lets move this into constructor

Reply:
I will allow for the whole BoxShadow in the constructor, gives more customization.